x/gov/keeper/constitution.go

```diff
@@ -0,0 +1,12 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (keeper Keeper) GetConstitution(ctx sdk.Context) (constitution string) {
+	store := ctx.KVStore(keeper.storeKey)
+	bz := store.Get([]byte("constitution"))
```

Comment:
Shouldn't we create a key for this under `x/gov/types/keys.go`

Reply:
Is this only meant to be in the genesis file, or do we also want to store it to state? It seems unnecessary to store such a big text on chain.

Reply:
My impression is that it must in exist in state, no?

Reply:
yes it needs to be in state. not all nodes have the genesis file meaning that if you don't have it you wont be able to query it